internal/pkg/cli/task_run.go

```diff
@@ -371,6 +379,42 @@ func (o *runTaskOpts) Validate() error {
 		}
 	}
 
+	if len(o.secrets) > 0 {
+		if err := o.validateSecrets(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *runTaskOpts) validateSecrets() error {
+	if o.appName == "" && o.env == "" && !o.acknowledgeSecretsAccess {
+		fmt.Println("Looks like you're requesting ssm:GetParameters to the following SSM parameters:")
```

Comment:
We should always use the `log` package here. 

```suggestion
		log.Infoln("Looks like you're requesting ssm:GetParameters to the following SSM parameters:")
```

`log` wraps the printing with the desired writer to write to. In this particular case, we don't want to print to stdout, but instead stderr.